clean up docs todos (#6886)
RasaHQ/rasa: docs/docs/connectors/your-own-website.mdx

### RestInput

The REST channel will provide you with a REST endpoint to post messages

> to and in response to that request will send back the bots messages.

Reviewer: ```suggestion
to and, in response to that request, will send back the bots messages.
```

> The REST channel will provide you with a REST endpoint to post messages
> to and in response to that request will send back the bots messages.

Author: ```suggestion
The REST channel will provide you with a REST endpoint where you can post
user messages and receive the bot's messages in response.
```

Author: Is this clearer?

Author: in response to https://github.com/RasaHQ/rasa/pull/6886#discussion_r499743854

Reviewer: Yes! Way clearer

### CallbackInput

The Callback channel behaves very much like the REST channel,
but instead of directly returning the bot messages to the HTTP
request that sends the message, it will call a URL you can specify
to send bot messages.

To use the callback channel, add the credentials to your `credentials.yml`:

```yaml-rasa
callback:
  # URL to which Core will send the bot responses
  url: "http://localhost:5034/bot"
```

Restart your Rasa X or Rasa Open Source server
to make the new channel endpoint available to receive messages. 
You can then send messages to `http://<host>:<port>/webhooks/callback/webhook`, replacing
the host and port with the appropriate values from your running Rasa X or Rasa Open Source server.

#### Request and Response Format

After making the `callback` input available, you can `POST` messages to
`http://<host>:<port>/webhooks/callback/webhook` with the following format:

```json
{
  "sender": "test_user",  // sender ID of the user sending the message
  "message": "Hi there!",
}
```

> The response will simply be `success`. Once Rasa Open Source is ready to send a

Reviewer: ```suggestion
If successful, the response will be `success`. Once Rasa Open Source is ready to send a
```
